resources/lang/sk.json

```
@@ -94,5 +94,7 @@
     "Reset": "Resetuj",
     "Reset the conversation": "Resetuj konverzáciu",
     "Are you sure you want to reset your conversation with Ester? It will start from the very beginning. This step can not be undone.": "Naozaj chceš resetovať tvoju konverzáciu s Ester? Začneš tak od úplného začiatku. Tento krok nejde vrátiť.",
-    "Your collection remains saved!": "Tvoja kolekcia zostáva uložená!"
+    "Your collection remains saved!": "Tvoja kolekcia zostáva uložená!",
+    "Location": "Lokácia",
+    "location_floor_formatted": ":floor podlažie"
```

I would keep the pattern to translate only english string to Slovak. Somethig like this:
To keep it simple
 
```suggestion
    "floor": "podlažie"
```

---

We're already using identifiers instead of natural language strings in About e.g. `help_title_1`.
I thought that this solution would be more abstract since you don't assume grammar rules of the language, but I'm okay with simplifying it.